python/pyspark/testing/utils.py

```diff
@@ -424,6 +425,11 @@ def assertDataFrameEqual(
     atol : float, optional
         The absolute tolerance, used in asserting approximate equality for float values in actual
         and expected. Set to 1e-8 by default. (See Notes)
+    returnUnequalRows: bool, False
```

Comment:
on second thought, I think we should rename this parameter to something else like `includeDiffRows` because technically it does not return but let the exception include the different rows. cc @allanf-db

Reply:
That makes sense to me. Updated the parameter name and following decriptions.

Reply:
Makes sense to me too, thanks